ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -262,6 +262,10 @@ def handle_new_superpixels(*args):
         def _invalidate_cache_on_sp_change(*args, **kwargs):
             op = opEdgeTrainingWithMulticut
             opEdgeTrainingWithMulticut.clear_caches(op.current_view_index())
+            # allow for dirty propagation to the volumina layers
```

Comment:
Do we need to do propagate dirtyness every time operator's caches are cleared? If yes, then this change should go to [`OpEdgeTrainingWithMulticut.clear_caches()`](
https://github.com/ilastik/ilastik/blob/068d8d709d74626e59a471bb50d9fcd607cecf27/ilastik/applets/edgeTrainingWithMulticut/opEdgeTrainingWithMulticut.py#L98).

Reply:
moving this change there would mean the operator itself would modify its slot (`FreezeCache`) which is not something that should happen.